src/Explore/ScoreIndicator.js

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react'
+
+import { Box } from '../Primitives'
+
+function ScoreIndicator({ score }) {
+  const [text, setText] = useState(score.toFixed(3))
```

Comment:
I would maybe store a boolean, like `const [helpVisible, toggleHelp] = useToggle(false);`, and then just `toggleHelp()` in the event handlers. The help text and score can then be moved to the JSX, in a ternary condition.